raft-tech/TANF-app
# chore(1415): bump django-admin-508 to 0.2.0

## Summary of Changes

This PR updates django-admin-508 to the latest version to address various fixes for visual inconsistencies in several Django Admin pages. This PR also disables the ability to add users via Django Admin, per @ADPennington's request in #1664.

Pull request closes #1415

## How to Test
_List the steps to test the PR_
These steps are generic, please adjust as necessary.
```
cd tdrs-frontend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
cd tdrs-backend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d 
```

1. Navigate to the Django Admin page and login.
1. If possible, compare the Django Admin page of this branch with the one deployed on staging
1. Ensure the changes visible in the screenshots below are present
> *Demo GIF(s) and screenshots for testing procedure*

![Screen Shot 2022-04-15 at 9 07 47 AM](https://user-images.githubusercontent.com/12901172/163621799-aeabbd9c-bda1-4221-ae94-ae2d88d1c04c.png)
![Screen Shot 2022-04-14 at 1 33 54 PM](https://user-images.githubusercontent.com/12901172/163621924-b505c56f-e714-4efa-9848-80afc821ea66.png)
![Screen Shot 2022-04-14 at 1 33 36 PM](https://user-images.githubusercontent.com/12901172/163621971-e1577bcb-6541-474b-86e3-99fffb05554e.png)

## Tabletop


## Deliverables
_More details on how deliverables herein are assessed included [here](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:

### [Deliverable 2: Tested Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
